examples/packet-production/cluster.lokocfg

```diff
@@ -57,6 +57,10 @@ cluster "packet" {
   cluster_name     = var.cluster_name
   controller_count = var.controllers_count
 
+  //os_channel       = "stable"
+  //os_version       = ""
```

Comment:
I think having `os_version` here, commented out and with empty value does not bring much value to those example configs. We should either remove it or add commented out `ipxe_script_url` in my opinion, but considering that using `ipxe_script_url` is not very well documented, I'd just remove `os_version` for now.

Reply:
done